src/internal/restapi/middleware/logreqid.go.tmpl

```diff
@@ -1,16 +1,15 @@
-package logreqid
+package middleware
 
 import (
-	"{{ .repoBaseUrl }}/{{ .serviceName }}/internal/repository/logging"
-	"{{ .repoBaseUrl }}/{{ .serviceName }}/web/filter/reqid"
 	"net/http"
 )
 
 func logRequestIdHandler(next http.Handler) func(w http.ResponseWriter, r *http.Request) {
 	handlerFunc := func(w http.ResponseWriter, r *http.Request) {
-		ctx := r.Context()
-		newCtx := logging.CreateContextWithLoggerForRequestId(ctx, reqid.GetRequestID(ctx))
-		r = r.WithContext(newCtx)
+		// example to log a request id
```

Comment:
This code won't work with logging removed when commenting in.

Let's restore it (including the manual logging) until we've discussed which logging framework to use in channel, it works for now, and then we can make the change to the template including a logging framework.

Reply:
Should we just remove this middleware component then? I'm not a fan of restoring functionality, which we definitely want to replace soon.

Reply:
- [ ] Restore logging component

Reply:
> Should we just remove this middleware component then? I'm not a fan of restoring functionality, which we definitely want to replace soon.

I'm with you on that one, it's just that "soon" has in the past turned out to be highly dependent on volunteer availability and priority of other tasks.